pom.xml

```diff
@@ -639,6 +659,13 @@
             <version>1.21</version>
         </dependency>
 
+        <!-- Needed by blueocean-rest-impl for ssh unititlies -->
+        <dependency>
+            <groupId>com.jcraft</groupId>
+            <artifactId>jsch</artifactId>
```

Comment:
Preferably would be shared with other copies via a library wrapper plugin.

Reply:
And in fact there is a `jsch` plugin! So use it.

Reply:
Yes, the blueocean-rest-impl should use jsch-plugin (library wrapper plugin)

Reply:
Done

Reply:
So why is this still here?